.github/workflows/cla.yaml

```diff
@@ -0,0 +1,14 @@
+name: cla-check
+
+on:
+  pull_request:
+  workflow_dispatch:
```

Comment:
This seems a bit odd... what would be checked?

Perhaps authors of the last NN commits?

Reply:
I was thinking it would be useful if it failed, then the contributor signed the CLA, and we could run it manually. But maybe we can just re-run the failed check, so that would be sufficient.